build.gradle

```diff
@@ -53,25 +71,27 @@ allprojects {
     }
 
     ext {
-        testcontainersVersion = "1.15.3"
-        slf4jVersion = "1.7.30"
+        testcontainersVersion = "1.16.0"
+        zstdVersion = "1.5.0-4"
```

Comment:
let's keep the same version as in kafka.

Reply:
Fixed